frontend/packages/core/src/card.tsx

```diff
@@ -1,17 +1,14 @@
 import * as React from "react";
 import styled from "@emotion/styled";
-import type {
-  CardContentProps as MuiCardContentProps,
-  CardHeaderProps as MuiCardHeaderProps,
-} from "@material-ui/core";
```

Comment:
I opted to use a plain div and not use the CardContent component b/c it has padding bugs w.r.t adding extra padding where it assumes the "card action" area will be. More details in this issue  https://github.com/mui-org/material-ui/issues/9871. Initially, I planned to override the component's last-child bottom padding to "0px" or use the `spacing` padding value but we/users might want the "card action area" to be different from the CardContent padding. The CardContent component itself is very simple so decided a div might be better so that we can control the padding more and we can handle the card action padding separately via the the CardAction component  https://material-ui.com/api/card-actions/ if we end up needing it for Dash. LMK what you think.

Reply:
I think it's fine especially since the card content renders a div anyway. You covered the only downside I can think of which is is we had card actions but I think we could handle that use case